src/index.html

```diff
@@ -22,6 +22,402 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- header -->
+    <header class="header">
+      <div class="logo">
+        <a href="https://snipboard.io/">
+          <img
+            src="images/logo.png"
+            alt="logo Moyo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="https://snipboard.io/"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/e"
+              data-qa="nav-hover"
```

Comment:
The `data-qa` attribute here is set to `nav-hover`, which is inconsistent with the task requirement to use `data-qa="card-hover"`. Please ensure consistency across all `data-qa` attributes.